plugins/inputs/sqlserver/sample.conf

```
@@ -19,6 +19,7 @@
   ## Authentication method
   ## valid methods: "connection_string", "AAD"
   # auth_method = "connection_string"
+  # user_assigned_id = "ClientID" # UserAssignedID is the client ID of the user assigned identity that should be used to authenticate to the Azure SQL server.
```

The config should show the default value and be below a description:

```suggestion
  # User assigned ID of the user assigned identity to authenticate to the SQL
  # server
  # user_assigned_id = "" 
```

Should this actually be called client ID? It is weird to call it user assigned ID, and then refer to it as the client ID.

---

Will rename it to client id